python/tvm/relay/build_module.py

```diff
@@ -192,6 +192,9 @@ def build(func, target=None, target_host=None, params=None):
 
     with tophub_context:
         bld_mod = BuildModule()
+        if "tvm.relay.module.Module" in str(type(func)):
```

Comment:
this is not the best way to check a type in python use

```
isinstance
```

Reply:
I used string comparison to avoid adding importing module.Module in build_module. I remember we did it in one of the prev PRs already.
Ok, I'll change it to use `isinstance` here.